test/tla/Annotations.tla

```diff
@@ -0,0 +1,37 @@
+-------- MODULE Annotations ----------
```

Comment:
Should it include annotations of functions?

Reply:
Yes, `FactFun` is a recursive function. It is often hard to distinguish functions from operators in TLA+